### ISSUE TITLE
the parser handling PLYROCK and PLYMAX incorrectly

### ISSUE THREAD

**Issue by GitPaean** [OPM/opm-parser#583]
Other keywords look fine from the POLY.inc in the opm-data. I have not spotted the exact reason, while it used to work when I changed the POLY.inc in opm-data about a week ago. 

The following is the error message. 
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  Number of columns in the data file isinconsistent with the ones specified
```

**Comment by joakim-hove** [comment]
OK - thanks; I will look into it. This is probably due to the table refactor work I have been doing lately - my bad :-(

**Comment by joakim-hove** [comment]
OK - cool if you give this: https://github.com/OPM/opm-parser/pull/584 a spin.

**Comment by GitPaean** [comment]
Fixed with PR OPM/opm-parser#584. Closing now.
